internals/daemon/response.go

```diff
@@ -169,22 +150,31 @@ func (f fileResponse) ServeHTTP(w http.ResponseWriter, r *http.Request) {
 	http.ServeFile(w, r, string(f))
 }
 
+// Reponsef behaves similar to the other printf style functions as long as additional
+// arguments beyond the format string are provided.
```

Comment:
The note below already states the formatting quirk. So I suggest we change this comment to what the function actually does. Something like:

```suggestion
// Responsef builds an error Response that returns the status and formatted message.
```

Reply:
Done, thanks.